homeassistant/components/idasen_desk/sensor.py

```diff
@@ -79,12 +79,18 @@ def __init__(
         self._attr_unique_id = f"{description.key}-{address}"
         self._attr_device_info = device_info
         self._address = address
+        self._desk = coordinator.desk
 
     async def async_added_to_hass(self) -> None:
         """When entity is added to hass."""
         await super().async_added_to_hass()
         self._update_native_value()
 
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self._desk.is_connected is True
```

Comment:
Good catch. It would probably be fine without it, since the coordinator updates always come from the desk callback, and not the other way around (there is no polling), but better have it now than miss it later.
I'll also do that for the cover. Let me know if you prefer that on a separate small PR instead.